Constitution.txt

```diff
@@ -1,490 +1,264 @@
-
-2: When vacancies happen in the representation of any State in the Senate, the executive authority of such State shall issue writs of election to fill such vacancies:  Provided, That the legislature of any State may empower the executive thereof to make temporary appointments until the people fill the vacancies by election as the legislature may direct. affects 4
-
-3: This amendment shall not be so construed as to affect the election or term of any Senator chosen before it becomes valid as part of the Constitution.   ratified #17
-
-Article [XVIII] (Amendment 18 - Prohibition of Intoxicating Liquors)16
-1: After one year from the ratification of this article the manufacture, sale, or transportation of intoxicating liquors within, the importation thereof into, or the exportation thereof from the United States and all territory subject to the jurisdiction thereof for beverage purposes is hereby prohibited.
-
-2: The Congress and the several States shall have concurrent power to enforce this article by appropriate legislation.
+Section 5
+1: These United States shall guarantee to every State in this Union a Republican Form of Government, and shall protect each of them against Invasion; and on Application of the Legislature, or of the Executive (when the Legislature cannot be convened) against domestic Violence.
 
-3: This article shall be inoperative unless it shall have been ratified as an amendment to the Constitution by the legislatures of the several States, as provided in the Constitution, within seven years from the date of the submission hereof to the States by the Congress.   ratified #18
+2: No new State shall be admitted to the Union, which has not established a written Constitution which shall serve as supreme Law in the State, subject not to any law passed by the Congress of these United States, but only to the Constitution of these United States, in which a Republican form of government shall be described; and which shall provide for the election of State Legislature, Representatives to the Congress of these United States, Electors for the Presidency of these United States, and a Chief Executive of the State, by the people of the State, or by their chosen representatives; and which shall provide for the determination of the Citizens of the State to remove themselves from the Union; and which shall recognize the Rights and Responsibilities of the People inherent in their being as are described in Article V; and which shall enshrine in law the duty of the State to protect these Rights, and not to infringe upon them except as punishment of an individual for crimes of which they have been convicted, by unanimous decision of a Jury of disinterested peers, in a court of Law; and which recognizes the State as the sole organization given the sanction to act in aggression in its territory, and then only in accordance with such Constitution and Laws as are approved by the People or their elected Representatives.
```

Comment:
Here I make clear that the Constitution of the Union is supreme over State Constitutions, but that the Congress cannot force the States to pass or revoke laws.

I also lay out a set of requirements for the State Constitutions, which form the skeleton of a Constitution which would be compatible with the Constitution of the Union. I make explicit reference to the next Article, which is annotated there.